components/class_loader.rst

```diff
@@ -7,6 +7,11 @@ The ClassLoader Component
     The ClassLoader component provides tools to autoload your classes and
     cache their locations for performance.
 
+.. caution::
+
+    The ClassLoader component was deprecated in Symfony 3.3 and it will be
```

Comment:
Technically, using the past tense here is not correct as 3.3 was released yet. Though I think that's okay as it allows us not to update the sentence in May again. What do you think?